### PR TITLE
dash/a11y-datagrid-sync-demo

### PR DESCRIPTION
**Pull request by Vazonik**
Improved accessibility in the Datagrid Sync demo.

### CODE REVIEW

**Review by karolkolodziej — APPROVED** [review]

File: samples/dashboards/demo/datagrid-sync/demo.js [hunk 1]
```diff
@@ -24,9 +21,29 @@ const chartOptions = {
...
+        connector values by manually changing the height of the bars in the 
+        series, which is possible with allowConnectorUpdate option set to true.`
+    },
+    tooltip: {
+        stickOnContact: true
+    },
+    xAxis: {
+        accessibility: {
+            description: 'categories of food',
+            rangeDescription: ''
```
Comment (sebastianbochan) [inline samples/dashboards/demo/datagrid-sync/demo.js:38]: Do we need it?

Comment (Vazonik) [reply samples/dashboards/demo/datagrid-sync/demo.js:38]: It seems to me that without this, the description of the axis is quite ambiguous, i.e. just "categories". Do you think this is enough or is it better to leave "categories of food"? @marvin19

Comment (marvin19) [reply samples/dashboards/demo/datagrid-sync/demo.js:38]: I do not think we need it in this case because the main point of this chart is showing features

**Review by marvin19 — CHANGES_REQUESTED** [review]
Looks good! I would also add unit to the tooltip, so that the user knows it is mcg and xAxis.accessibility.description can be removed.

**Review by sebastianbochan — APPROVED** [review]

**Review by marvin19 — APPROVED** [review]

**Pull request merged**